src/components/PreviewComment/index.js

```diff
@@ -72,7 +72,7 @@ const PreviewComment = ({comment, time, image, totalComment, onPress, user, item
             <View style={styles.containerUsername}>
               <CommentUserName isPreviewComment comment={item} user={user} />
               <Gap width={4} />
-              <Dot size={4} color={COLORS.blackgrey} />
+              <Dot size={4} color={'#828282'} />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The hardcoded color '#828282' should be replaced with a constant for better maintainability.
```diff
- <Dot size={4} color={'#828282'} />
+ <Dot size={4} color={colors.grey} />
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->